packages/scss/src/components/_action-icon.scss

```diff
@@ -1,7 +1,7 @@
 .actionIcon {
 	background: transparent;
 	border: 0;
-	border-radius: 50%;
+	border-radius: _component("textfield.border-radius");
```

Comment:
It's weird to use a radius coming from another component for no reason.
Why not use either the commons.border.radius or define a specific actionIcon border radius ?